package.json

```diff
@@ -3567,22 +3568,33 @@
         "@iarna/toml": "^2.2.5",
         "adm-zip": "^0.5.10",
         "amazon-states-language-service": "^1.10.0",
+        "assert": "^2.0.0",
         "async-lock": "^1.4.0",
         "aws-sdk": "^2.1384.0",
         "aws-ssm-document-language-service": "^1.0.0",
+        "browserify-fs": "^1.0.0",
+        "browserify-zlib": "^0.2.0",
```

Comment:
It will be hard to reason about which of these interleaved dependencies is needed for which "mode". Can we create a sub-project in a new folder like `srcWeb/` which has its own package.json? 

Example of a project "inheriting" a parent project: https://github.com/aws/aws-toolkit-common/blob/147df8f44f08e081675e01cebf9f957ca9658add/lsp/core/aws-lsp-yaml-common/package.json#L10

Reply:
I've created a new `package.json` with the browser dependencies in `src/browser/`. This should be good enough for now to separate things, but when it comes to naming we can revisit later and come to a final agreement.